source/scss/components/_linked_price.scss

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
File name should have a '-' instead of a '_' in the middle ("linked-price.scss")